src/file_writer.rs

```diff
@@ -0,0 +1,23 @@
+extern crate calamine;
+
+use crate::calculation::Out;
+use std::fs::File;
+use std::io::Write;
+
+pub fn write_file(out: Out, out_file: &mut File) {
+    writeln!(
+        out_file,
+        "{}|{}|{}|{}|{}|{}|{}|{}|{}|{}",
+        out.id,
+        out.from_year,
+        out.from_months,
+        out.from_days,
+        out.to_year,
```

Comment:
this code is correct but i want you  to read about implementing fmt::Display for a struct...and implement it here....the benefit of doing that is you can directly print the struct and you wont have to type this much.....try to find out if u could not find it ask me...I'll tell u how...i am not expecting that u will know this but want u to learn...so do your research and call me if u have trouble understanding